manifests/n/nostar/DroidStar/f050489/nostar.DroidStar.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created with Komac v1.9.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: nostar.DroidStar
+PackageVersion: f050489
+InstallerLocale: en-US
+InstallerType: burn

```

Comment:
```suggestion
InstallerType: portable
```

Reply:
Why?

Reply:
Because it is not an installer - it is a portable app. Or at least I think it is.